### DISCUSSION TITLE
extensions load url / in local

### DISCUSSION THREAD

**Discussion by Eaielectronic** [TurboWarp/packager#913] [Ideas]
it would be good if the Turbowarp extensions were not loaded from a url but from the html file (make an option to load the extensions by url or put it in the html file like custom extensions)

**Comment by GarboMuffin** [comment]
they already get downloaded once and embedded in the html file

**Comment by Eaielectronic** [comment]
I don't know, when I turn off the WiFi, an error message appears on loading saying that the extension could not be loaded from its URL (for official TurboWarp extensions like Key Simulation or JSON).
